lib/handlers/handler.js

```diff
@@ -11,6 +11,7 @@ function handler(info, params, res) {
       res.json({
         ok: false,
         error: (error && error.message) || error,
+        details: (error && error.details) || undefined,
```

Comment:
It does still render the property; in the Koa codebase I tried to be smart about it and only add the property when details were available but it proved to be difficult because the response was a setter in that codebase, so I opted to include it always, but sometimes be undefined. If you think it would be better to refactor into something like the following let me know.

```
if (error && error.details) {
  res.json({
         ok: false,
         error: (error && error.message) || error,
         details: (error && error.details) || undefined,
    });
  } else {
    res.json({
         ok: false,
         error: (error && error.message) || error,
    })
}
```